docs/semgrep-code/triage-remediation.md

```diff
@@ -138,7 +138,7 @@ Triage your Semgrep Cloud Platform findings displayed as comments in GitHub PRs
 
 :::info Prerequisites
```

Comment:
```suggestion
```

Reply:
We don't need this callout, as the sidebar reorg makes it clear that you should finish a core deployment first.